dart/generator.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:collection';
+
```

Comment:
Unfortunately, iterables in dart suck - there's no easy way to define unlimited iterators without reinventing the wheel. It's even more verbose than Rust!